app/templates/profile/profile_main.html

```diff
@@ -0,0 +1,15 @@
+{% extends "layout.html" %}
+
+{% block content %}
+
+  <div class="profile-info">
+    {% block profilecontent %}{% endblock %}
+  </div>
+
+{% endblock %}
+
+{% block rightblock %}
+  <div class="profile-side-info">
+    {% block profileSideContent %}{% endblock %}
+  </div>
```

Comment:
Pushed the `rightcontent` styling down the tree. Now, I'm less restricted while working on the blogpost stuff, but our general layout is consistent.